.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Description of the changes
+
+_In this section, explain **what and why** not how._
+
+_In most cases, you can leave out details about how a change has been made. Code is generally self-explanatory in this regard (and if the code is so complex that it needs to be explained in prose, that’s what source comments are for). Just focus on making clear the reasons why you made the change in the first place, the way things worked before the change (and what was wrong with that), the way they work now, and why you decided to solve it the way you did._
+
+## Breaking change
+
+- [ ] No
+- [ ] Yes (in the Helm chart(s)): _indicate the chart, version & release note link_
+- [ ] Yes (in the module itself): _give an explanation of the breaking change_
```

Comment:
This module does not embed any Helm chart, so one of the yes lines can be removed.

```suggestion
## Breaking change

- [ ] No
- [ ] Yes
```